### PR TITLE
Created organic matter.json and apparently never updated the regex strings so they acutally work

### PR DESCRIPTION
**Pull request by Su386yt**

### CODE REVIEW

File: data/constants/organic_matter.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (j10a1n15) [inline data/constants/organic_matter.json:1]: FLOWERING_BOUQUET - 6k

**Review by j10a1n15 — CHANGES_REQUESTED** [review]

File: data/constants/organic_matter.json (hunk 1 above)
Comment (Su386yt) [reply data/constants/organic_matter.json:1]: https://hypixel-skyblock.fandom.com/wiki/Compost

Comment (j10a1n15) [reply data/constants/organic_matter.json:1]: https://hypixel-skyblock.fandom.com/wiki/Composter 
image


https://wiki.hypixel.net/Composter
image

Comment (Su386yt) [reply data/constants/organic_matter.json:1]: Fxied

**Pull request merged**